examples/ml/housing-prices/README.md

```diff
@@ -0,0 +1,247 @@
+
+### Evaluate the model
+After the model is trained we output some visualizations to evaluate its effectiveness of it using the learning curve and other statistics.
+<p align="center">
+  <img src="images/cv_reg_output-1.png">
+</p>
+
+
+## Pachyderm Pipeline
+Now we'll deploy this python code with Pachyderm. 
+### TLDR; Just give me the code
+
+```bash
+# Step 1: Create input data repository
+pachctl create repo housing_data
+
+# Step 2: Create the regression pipeline
+pachctl create pipeline -f regression.json
+
+# Step 3: Add the housing dataset to the repo
+pachctl put file housing_data@master:housing-simplified.csv -f data/housing-simplified-1.csv
+
+# Step 4: Download files once the pipeline has finished
+pachctl get --recursive regression@master
+
+# Step 5: Update dataset with more data
+pachctl put file housing_data@master:housing-simplified.csv -f data/housing-simplified-2.csv --overwrite
+```
+
+### Step 1: Create input data repository
+
+Once the pachyderm cluster is running, we will create a data repository called `housing_data` where we will put our dataset.
+
+```bash
+$ pachctl create repo housing_data
+$ pachctl list repo
+NAME                CREATED             SIZE
+housing_data        3 seconds ago       0 B
+```
+
+### Step 2: Create the regression pipeline
+
+We can now connect a pipeline to watch the data repo. Pipelines are defined in `json` format. Here is the one that we'll be used for the regression pipeline: 
+```json
+# regression.json
+{
+    "pipeline": {
+        "name": "regression"
+    },
+    "description": "A pipeline that trains produces a regression model for housing prices.",
+    "input": {
+        "pfs": {
+            "glob": "/*",
+            "repo": "housing_data"
+        }
+    },
+    "transform": {
+        "cmd": [
+            "python",
+            "regression.py",
+            "--input",
+            "/pfs/housing_data/",
+            "--target-col",
+            "MEDV",
+            "--output",
+            "/pfs/out/"
+        ],
+        "image": "jimmywhitaker/housing-prices"
+    }
+}
+```
+
+For the pipeline **input** we can define what data repo(s) our pipeline is connected to and what [glob pattern](https://docs.pachyderm.com/latest/concepts/pipeline-concepts/datum/glob-pattern/) to use. A glob pattern tells the pipeline how to map data into a job, here we have it create a new job for each datum in the `housing_data` repository.
+
+The **image** defines what Docker image will be used for the pipeline, and the **transform** is the command run once a pipeline job starts.
+
+Once this pipeline is created, it will be looking for any changes to its input, if detected it will start a new job to train given the new dataset.
+
+```bash
+$ pachctl create pipeline -f regression.json
```

Comment:
we don't use the $ sign in the command blocks.

Reply:
Corrected.

Reply:
I think the lack of a new command indicator is a little difficult to read in code blocks sometimes. For example, [here](https://github.com/pachyderm/pachyderm/blob/example/housing-prices/examples/ml/housing-prices/README.md#step-6-inspect-the-pipeline-lineage). What do you recommend?